foxpuppet/windows/browser/window.py

```diff
@@ -12,7 +12,8 @@
 
 class BrowserWindow(BaseWindow):
 
-    """Representation of a browser window."""
+    """Representation of a browser window.
```

Comment:
Yes, at the moment the generated docs do not detail the parameters for constructing a `BrowserWindow` object.